Xiyuan_Guan_Exercise08.R

```diff
@@ -0,0 +1,59 @@
+# BioComputing Exercise 08
+# 2022-11-04
+# Written by Xiyuan Guan
+
+rm(list = ls())
+gc()
+
+setwd("C:/Users/guanx/Documents/N.ND/BIOS 60318 Biocomputing/test/exercise8/Exercise08/")
+
+# task 1
+# Write R code that replicates the functionality of the head function we used in bash.
+# define a variable with the file to return lines from and a variable representing the number of lines to be returned from the top of the indicated file. 
+# The selected file content should be printed to the terminal in R
+# Do not use the head function in R. 
+# Be sure you are using square bracket indexing to subset the tabular file.
+
+testfile<- read.csv(file = "iris.csv", header = T) # load test file.
+
+head2 <- function(f, line){
+  subf <- f[1:line,]
+  return(subf)
+}
+# Usage: head2(file to return lines from, the number of lines to be returned from the top of the indicated file)

```

Comment:
The question does not ask you to write a function in R but codes to replicate a function
-0.25

Reply:
We have not learnt how to make a function in R yet